cmd/root.go

```diff
@@ -320,13 +320,18 @@ func AddFlags(flagSet *pflag.FlagSet, cfg *config.Config) (configPath *string) {
 	/**======================== PoET Flags ========================== **/
 
 	flagSet.DurationVar(&cfg.POET.PhaseShift, "phase-shift",
-		cfg.POET.PhaseShift, "phase shift of poet server")
+		cfg.POET.PhaseShift, "phase shift of poet server: duration after epoch start, at which poet round starts")
 	flagSet.DurationVar(&cfg.POET.CycleGap, "cycle-gap",
-		cfg.POET.CycleGap, "cycle gap of poet server")
+		cfg.POET.CycleGap, "cycle gap of poet server: gap between poet rounds")
 	flagSet.DurationVar(&cfg.POET.GracePeriod, "grace-period",
-		cfg.POET.GracePeriod, "time before PoET round starts when the node builds and submits a challenge")
-	flagSet.DurationVar(&cfg.POET.RequestTimeout, "poet-request-timeout",
-		cfg.POET.RequestTimeout, "timeout for poet requests")
+		cfg.POET.GracePeriod, "time before poet round starts, when the node builds and submits a challenge")
+	flagSet.DurationVar(&cfg.POET.SubmitChallengeTimeout, "submit-challenge-timeout",
+		cfg.POET.SubmitChallengeTimeout,
+		"time within grace period, when post challenge must be submitted to poet server")
+	flagSet.DurationVar(&cfg.POET.FetchProofTimeout, "get-proof-timeout",
+		cfg.POET.FetchProofTimeout, "time period when ready poet proof must be fetched")
```

Comment:
```suggestion
	flagSet.DurationVar(&cfg.POET.FetchProofTimeout, "get-proof-timeout",
		cfg.POET.FetchProofTimeout, "timeout for fetching a poet proof")
```

Reply:
Removed FetchProofTimeout and SubmitChallenge timeouts at all because of overengineering: RequestTimeout is enough for GetProof requests and to submit challenge there is no special timeout is needed